Opcodes/OpcodeBase.hpp

```diff
@@ -374,7 +379,7 @@ class OpcodeNoteoffBase
 };
 
 #if (__cplusplus >= 201103L) && !(defined(__wasi__))
-#pragma message("Defining heap_object_manager_t.")
+// #pragma message("Defining heap_object_manager_t.")
```

Comment:
Whats the warning here?

Reply:
Oops, I thought I had seen somewhere that pragma messages count as warnings, but I guess I was wrong, so I reverted.